storage/src/block_meta_storage.rs

```diff
@@ -427,26 +420,12 @@ impl Decoder for Meta {
                         let block_hash = bytes
                             [next_successor_index..(next_successor_index + LEN_BLOCK_HASH)]
                             .to_vec();
-                        assert_eq!(
```

Comment:
This should newer be triggered as the resulting length is  `next_successor_index + LEN_BLOCK_HASH-next_successor_index` that is `LEN_BLOCK_HASH`, and also correct length is checked below in `BlockHash::try_from(block_hash)?`.